scripts/test_binary.py

```diff
@@ -61,6 +61,22 @@
             print("Some vs == 0, {} incomplete".format(args.process_type))
         sys.exit(1)
 
+    # binary zero check
+    # Checks 10 random stations for any occurances of 0 in the output (aka results have not been written)
+    for i in range(10):
```

Comment:
possible simplification (imo) if you want:
```python
for station_name in np.random.choice(bin.stations.name, replace=False):  
    acc = bin.acc(station_name)
```

Assumes that `bin.stations.name` is a list or array